packages/suite/src/components/wallet/WalletLayout/AccountTopPanel/AccountTopPanel.tsx

```diff
@@ -74,20 +78,28 @@ export const AccountTopPanel = () => {
         );
     }
 
-    const { symbol, formattedBalance } = account;
+    const { symbol, formattedBalance, index, accountType } = account;
 
     return (
         <AppNavigationPanel
             maxWidth="small"
             title={
                 <MetadataLabeling
-                    defaultVisibleValue={<AccountLabeling account={account} />}
+                    defaultVisibleValue={
+                        <AccountLabel
```

Comment:
`AccountLabeling` is heavy component that handles cross-device/wallet accounts. Here it is not needed as the header account label is always without the device/wallet prefix

Reply:
`AccountLabel` is just for the account label, combines the existing label with the default one and adds some styling if needed (`TabularNums`).

It uses `defaultAccountLabelString` under the hood to (from hook) to get the defualt value. Same as it is used here for `defaultEditableValue`.